Common/Global.cs

```diff
@@ -23,6 +25,33 @@
 
 namespace QuantConnect
 {
+    public enum PrimaryExchange : byte
+    {
+        UNKNOWN=32,
```

Comment:
Curious why just not 0, empty char vs `' '` space char?
Let's add some xml doc for the new enum 👍

Reply:
>Curious why just not 0, empty char vs ' ' space char?
Good question, I think it's was more a matter of taste, the closes we can get to a empty char is `'\0'`, the `null char`.
Seems to me that space char was more clear than the null char as default value for the  `MapFileRow `constructor.